1.7hm.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#define maxAvailableNumber 100000
+
+int main() 
+{
+    int numbers[maxAvailableNumber] = { 0 };

```

Comment:
```suggestion
    int primeNumbers[maxAvailableNumber] = { 0 };
```
Более говорящие имена переменных.